.circleci/config.yml

```
@@ -569,3 +609,22 @@ workflows:
       #     <<: *push_builds_only
       #     requires:
       #       - 'Nomodule Build (Test)'
+
+  'Release':
+    jobs:
+      - initialize_repository:
+          name: 'Initialize Repository'
+          <<: *release_builds_only
+      - amp_release:
+          name: '<< matrix.flavor >> << matrix.esm >>'
+          matrix:
+            parameters:
+              flavor: ['base', 'experimentA', 'experimentB', 'experimentC']
+              esm: ['no-esm', 'esm']
+          <<: *release_builds_only
+          requires:
+            - 'Initialize Repository'
+      - archive_release:
+          name: 'Archive Release'
+          requires:
+            - amp_release
```

nit: For consistency, probably best to fully specify the conditions for each job.
```suggestion
          <<: *release_builds_only
          requires:
            - amp_release
```

---

Ha, somehow missed this